src/tests/test_common/homestore_test_common.hpp

```diff
@@ -28,6 +28,7 @@
 #include <homestore/index_service.hpp>
 #include <homestore/replication_service.hpp>
 #include <homestore/checkpoint/cp_mgr.hpp>
+#include <../lib/device/hs_super_blk.h>
```

Comment:
I don't think we need to do give relative path, since lib is always included, can we do `#include "device/hs_super_blk.h"?

Reply:
yes, it should work